thundra/integrations/botocore.py

```diff
@@ -205,15 +223,14 @@ def getRequestType(self, string):
         return Constants.AWS_SERVICE_REQUEST
 
     def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                 exception):
+                         exception):
 
         operation_name, request_data = args
         self.streamName = request_data['StreamName']
 
         scope.span.domain_name = Constants.DomainNames['STREAM']
         scope.span.class_name = Constants.ClassNames['KINESIS']
-        scope.span.operation_name = self.getRequestType(operation_name)
-
+        scope.span.operation_name = self.streamName
```

Comment:
what if there is no `streamName` in the request? For example, when the request is for list streams